1-exercises/A-accessing-values/exercise2.js

```diff
@@ -17,7 +17,7 @@ let capitalCities = {
 */
 
 let myCountry = "UnitedKingdom";
-let myCapitalCity; // complete the code
+let myCapitalCity ="london"; // complete the code
```

Comment:
Issue: The goal here was to select the capital from the `capitalCities` object.

For example using `capitalCities[myCountry]` will access the member of the object based on the value stored in `myCountry`